roles/job/tasks/main.yml

```diff
@@ -74,20 +74,6 @@
     - k8s_job['resources'] is defined
     - (k8s_job["resources"]|length>0)
 
-- name: Read Secret Configuration
```

Comment:
Since this is a required field, these tasks are not needed, which saves on reconciliation time.